src/createServer.js

```diff
@@ -1,3 +1,42 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const fs = require('fs');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const fileName = req.url.slice(1);
```

Comment:
This line slices the URL at the first character which is not necessary. You can directly get the pathname from urlParams which you created on the next line.